internal/tasks/tasks.go

```diff
@@ -240,6 +244,22 @@ func ExecuteTask(task Task) Task {
 			taskResult := taskDisableTunnel()
 			task.Result = sql.NullString{String: taskResult, Valid: true}
 
+		case "start_shell":
+			log.Println("Starting SSHX.io Shell")
+			var args taskStartShellArgs
+			err := json.Unmarshal([]byte(task.Args.String), &args)
+			if err != nil {
+				log.Printf("Error reading arguments or start_shell task: %s", err)
```

Comment:
Do we need to wrtite back the status of the job?

Reply:
Good catch. Gotta test it going through the error branch, will get back to you on this later ;)